### PR TITLE
[TASK] Change SSH include because other file should be unique created

### PR DESCRIPTION
**Pull request by Cyb10101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,14 @@ services:
 
     volumes:
       - .:/app
-      - ~/.ssh:/home/application/.ssh
+      - ~/.ssh/config:/home/application/.ssh/config:ro
```
Comment (ChrisB9) [inline README.md]: i don't like having mapped that many volumes into my container and i don't think thats wise performance-wise

File: ssh-agent.plugin.zsh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# test if agent-forwarding is enabled
+zstyle -b :omz:plugins:ssh-agent agent-forwarding _agent_forwarding
+
+if [[ $_agent_forwarding == "yes" && -n "$SSH_AUTH_SOCK" ]]; then
+	# Add a nifty symlink for screen/tmux if agent forwarding
+	[[ -L $SSH_AUTH_SOCK ]] || ln -sf "$SSH_AUTH_SOCK" /tmp/ssh-agent-$USER-screen
+elif [[ -f "$_ssh_env_cache" ]]; then
+	# Source SSH settings, if applicable
+	. $_ssh_env_cache > /dev/null
+	if [[ $USER == "root" ]]; then
```
Comment (ChrisB9) [inline ssh-agent.plugin.zsh]:
```suggestion
        FILTER="x"
	if [[ $USER == "root" ]]; then
	    FILTER="ax"
	fi
```

**Comment by ChrisB9** [comment]
could you split the ssh-agent plugin and the entrypoint because the entrypoint part looks good, i, though, have to look deeper into the zsh plugin and therefor want to take myself some time

File: README.md (hunk 1 above)
Comment (Cyb10101) [reply README.md]: Removed

File: ssh-agent.plugin.zsh (hunk 2 above)
Comment (Cyb10101) [reply ssh-agent.plugin.zsh]: changed

**Comment by Kanti** [comment]
@Cyb10101 can you clarify that this PR fixes or improve?

**Comment by Kanti** [comment]
@Cyb10101 I will close this PR. If you want to explain yourself, feel free to reopen it.

**Pull request closed without merging**